database/migrations/2021_05_13_213244_bump_version040301.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use App\Models\Configs;
+use Illuminate\Database\Migrations\Migration;
+
+class BumpVersion040301 extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Configs::where('key', 'version')->update(['value' => '040301']);
```

Comment:
This probably will need to be a `040302` as there is the other PR pending. ;)

Reply:
Whatever PR is merged first, I will re-base the other PR on the updated master and then re-run `make minor_version` again to avoid conflicts.

Reply:
:+1: